talent/onboarding/Non-EU_residents.md

```diff
@@ -1,128 +1,142 @@
-# Candidates from outside of the European Union
-This is the hardest process and we need the approval from the Spanish Government. We have to get the work permit and the visa in the passport. The first goal, the work permit, is the responsibility of the Company and the process must take place in Spain. The second goal, the visa in the passport, is the responsibility of the candidate and must take place in the Spanish Consulate in the country where the candidate is living. 
+# Candidates from Outside the European Union
```

Comment:
maybe just add: who are relocating to Spain

Reply:
Makes sense, done.